home/web/client/components/AccountsChooser/AccountsChooser.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import DropDownMenu from 'material-ui/DropDownMenu';
 import MenuItem from 'material-ui/MenuItem';
 
-import isEqual from 'lodash.isequal';
```

Comment:
Any links that confirm the `only what's used is included in builds`? I'm pretty sure it's not the case (until we get webpack2 and lodash get's rewritten)